src/ui/bind_handlers.js

```diff
@@ -38,8 +42,13 @@ export default function bindHandlers(map: Map, options: {}) {
     DOM.addEventListener(el, 'mouseup', onMouseUp);
     DOM.addEventListener(el, 'mousemove', onMouseMove);
     DOM.addEventListener(el, 'mouseover', onMouseOver);
-    DOM.addEventListener(el, 'touchstart', onTouchStart, {passive: false});
-    DOM.addEventListener(el, 'touchmove', onTouchMove, {passive: true}); // `passive: true` because onTouchMove only sends a map event.
+    // Bind touchstart with passive: true because onTouchStart only fires a map event
+    DOM.addEventListener(el, 'touchstart', onTouchStart, {passive: true});
+    // Bind touchmove with passive: false on iOS because, even though
+    // onTouchMove only fires a map event, binding with passive: true causes
+    // iOS not to respect e.preventDefault() in _other_ handlers, even if they
+    // are non-passive.  https://bugs.webkit.org/show_bug.cgi?id=182521
```

Comment:
Is this the right WebKit bug URL?

Reply:
Whoops, I must have copied from the wrong tab, thanks.